WooCommerce/Classes/ViewModels/Feature Announcement Cards/FeatureAnnouncementCardViewModel.swift

```diff
@@ -32,12 +33,14 @@ class FeatureAnnouncementCardViewModel {
         config.dismissAlertMessage
     }
 
-    private(set) var shouldBeVisible: Bool = false
+    @Published private(set) var shouldBeVisible: Bool = false
 
     init(analytics: Analytics,
-         configuration: Configuration) {
+         configuration: Configuration,
+         stores: StoresManager = ServiceLocator.stores) {
         self.analytics = analytics
         self.config = configuration
+        self.stores = stores
```

Comment:
Exactly, good catch! 👏 We want to test what happens when the Upsell Card Readers banner is/ is not visible, and that is discerned by this class. Please note that in this case, we didn't mock the `FeatureAnnouncementCardViewModel` itself, but just passed the stores mock where we can change the conditions. We could also mock the `FeatureAnnouncementCardViewModel` to make us able to decide the banner visibility, but since we already had this setup, I skip it because it would add an extra code without adding much value.